### PR TITLE
renaming and re-organizing openapi spec files

### PR DESCRIPTION
**Pull request by hmmtrifork**

### CODE REVIEW

File: openapi/user/time-registration.yaml [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
   tags:
-  - "Time registration"
+  - "TimeRegistration"
   operationId: "getTimeRegistrationsForUser"
-  description: "Returns time registrations for a user"
+  description: "Returns time registrations that belongs to a user"
   responses:
     "200":
-      description: "List of time registrations for the user"
+      description: "List of time registrations for a user"
```
Comment (joratrifork) [inline openapi/user/time-registration.yaml:8]: `a` (any) conflicts with the above description (specific):
> Returns time registrations that belongs to a user

So maybe revert this line?

File: openapi/trireg2admin.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+openapi: "3.0.3"
+info:
+  title: "TriReg2 Admin API"
+  description: "An OpenAPI specification for time registrations to be used by an administrator"
+  version: "0.1.2"
```
Comment (joratrifork) [inline openapi/trireg2admin.yaml:5]: Maybe a more significant bump in version number(s) due to significant breaking changes?

**Review by joratrifork — APPROVED** [review]
Don't merge until the Kotlin part has been approved (about to be merged), as this will likely cause CI actions to fail on active pull requests

File: openapi/schemas/Status.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+type: "string"
+description: "Describes the status of the time registration"
+nullable: true
+enum:
+  - "PENDING"
+  - "VALID"
+  - "INVALID"
+  - null
```
[the remark this answers is not in the document]
Comment (hmmtrifork) [reply openapi/schemas/Status.yaml:8]: They write that null value must be present in the list of enums https://swagger.io/docs/specification/data-models/enums/

File: openapi/user/time-registration.yaml (hunk 1 above)
Comment (hmmtrifork) [reply openapi/user/time-registration.yaml:8]: will do. Good point

File: openapi/trireg2admin.yaml (hunk 2 above)
Comment (hmmtrifork) [reply openapi/trireg2admin.yaml:5]: good point. will change to v0.2.0

**Comment by hmmtrifork** [comment]
> Don't merge until the Kotlin part has been approved (about to be merged), as this will likely cause CI actions to fail on active pull requests

I will wait until the Kotlin part is ready.

**Pull request merged**